tests/test_run_subproc.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+import builtins
+
+import pytest
+
+from xonsh.platform import ON_WINDOWS
+from xonsh.built_ins import run_subproc
+
+from tools import skip_if_on_windows
+
+
+@pytest.yield_fixture(autouse=True)
+def chdir_to_test_dir(xonsh_builtins):
```

Comment:
I see. I do have some unpushed fixes to the test suite. But I think it should wait until this goes in.